static/app/views/llmMonitoring/landing.tsx

```diff
@@ -25,13 +25,20 @@ import * as ModuleLayout from 'sentry/views/performance/moduleLayout';
 import {ModulePageProviders} from 'sentry/views/performance/modulePageProviders';
 import {ModulesOnboarding} from 'sentry/views/performance/onboarding/modulesOnboarding';
 import {OnboardingContent} from 'sentry/views/performance/onboarding/onboardingContent';
+import {useHasDataTrackAnalytics} from 'sentry/views/performance/onboarding/useHasDataTrackAnalytics';
 import {useModuleBreadcrumbs} from 'sentry/views/performance/utils/useModuleBreadcrumbs';
 
 export function LLMMonitoringPage() {
   const organization = useOrganization();
 
   const crumbs = useModuleBreadcrumbs('ai');
 
+  useHasDataTrackAnalytics(
+    new MutableSearch('span.category:"ai"'),
+    'api.performance.ai.landing-llm-monitoring-onboarding',
```

Comment:
Another nit, up to you whether you want to change this or not:

I noticed that for all these referrers, we're referring to them as `<module>-onboarding`, but not sure how they relate to onboarding? Should they be `<module>-onboarding` instead?